pom.xml

```diff
@@ -35,6 +35,7 @@
         <commons-codec.version>1.10</commons-codec.version>
         <jackson.version>2.10.0.pr1</jackson.version>
         <junit.version>4.12</junit.version>
+        <json-simple.version>1.1.1</json-simple.version>
```

Comment:
we already import jackson-databind in this pom.xml
that lib supports object -> json serialization via ObjectMapper
i'm not sure we need another json lib, unless this one does something that jackson doesnt?

https://fasterxml.github.io/jackson-databind/javadoc/2.10/com/fasterxml/jackson/databind/ObjectMapper.html?is-external=true

Reply:
Nice thx, I'll look into using that library

Reply:
but I really liked that package!